orte/util/proc_info.c

```diff
@@ -213,13 +227,14 @@ int orte_proc_info(void)
         }
         /* if we didn't find a match, then just use the hostname as-is */
         if (!match) {
-            orte_process_info.nodename = strdup(hostname);
+            orte_process_info.nodename = strdup(local_hostname);
         }
         opal_argv_free(prefixes);
     } else {
-        orte_process_info.nodename = strdup(hostname);
+        orte_process_info.nodename = strdup(local_hostname);
     }
 
+    printf( "local_hostname after assignment and free = %s\n", local_hostname);
```

Comment:
Not sure if this printf was intended to be here or not, but it probably shouldn't be in the final commit.

Reply:
also oops, should be fixed.